index.js

```diff
@@ -1,41 +1,88 @@
 import assert from 'node:assert'
 import ms from 'ms'
 
-export class CancelStuckTransactions {
+export const cancelTx = ({
+  tx,
+  recentGasUsed,
+  recentGasFeeCap,
+  log,
+  sendTransaction
+}) => {
+  // Increase by 25% + 1 attoFIL (easier: 25.2%) and round up
+  const maxPriorityFeePerGas = (tx.maxPriorityFeePerGas * 1252n + 1000n) / 1000n
+  const gasLimit = Math.ceil(recentGasUsed * 1.1)
+
+  log(`Replacing ${tx.hash}...`)
+  log(`- maxPriorityFeePerGas: ${tx.maxPriorityFeePerGas} -> ${maxPriorityFeePerGas}`)
+  log(`- gasLimit: ${recentGasUsed} -> ${gasLimit}`)
+  return sendTransaction({
+    to: tx.from,
+    value: 0,
+    nonce: tx.nonce,
+    gasLimit,
+    maxFeePerGas: maxPriorityFeePerGas > recentGasFeeCap
+      ? maxPriorityFeePerGas
+      : recentGasFeeCap,
+    maxPriorityFeePerGas
+  })
+}
+
+export const getRecentSendMessage = async () => {
```

Comment:
It would be great to add JSDoc comments to describe the return type of this function to enable IDE auto-completion for users of this module. It's fine to do that in a follow-up PR.

Reply:
I'm happy to accept PRs :)

Reply:
I personally don't use features like this